### PR TITLE
Merge/development frontend

### PR DESCRIPTION
**Pull request by ephraimeg**
Issue #14 
Issue #4

### CODE REVIEW

File: client/pages/login.vue [hunk 1]
```diff
@@ -40,8 +40,8 @@ export default {
   },
 
   methods: { 
-    login() { 
-      axios.post("http://192.168.1.2:8000/login", { 
+    login() {
+      axios.post("http://localhost:8000/login", {
```
Comment (aldnav) [inline client/pages/login.vue]: Find a way to not hardcode the endpoint here. Perhaps you can look at [`process.env`](https://nodejs.org/api/process.html#process_process_env).

Expected:

```js
axios.post(`${process.env.SERVER_URL}/login`, {
```

Comment (aldnav) [reply client/pages/login.vue]: @boredchinese @ephraimeg

**Comment by ephraimeg** [comment]
@boredchinese will make a new PR to merge front end to development

File: client/pages/login.vue (hunk 1 above)
Comment (ephraimeg) [reply client/pages/login.vue]: changes made [here](https://github.com/ephraimeg/alive-project/blob/ec966d22da303a304b4587eb6804e35cfcdcb975/client/src/pages/Login.vue)

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (aldnav) [inline .gitignore]: This is a conflict you need to resolve

File: client/.env [hunk 3]
```diff
@@ -0,0 +1 @@
+API_URL=http://192.168.1.2:8000
```
Comment (aldnav) [inline client/.env]: Is this supposed to be tracked?

File: client/src/pages/Appeal/Create.vue [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+      console.log(config)
+
+      axios.post(`${process.env.API_URL}/request/`, {
+        request_title: this.request_title,
+        detail: this.detail
+      }, config)
+
+      .then((res) => { 
+        console.log("NICE")
```
Comment (aldnav) [inline client/src/pages/Appeal/Create.vue]: What is supposed to be done here?

File: client/src/pages/Login.vue [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+    methods: { 
+        login() {
+            axios.post(`${process.env.API_URL}/login`, {
+                username: this.username,
+                password: this.password
+            })
+
+            .then((res) => { 
+                if(res.data.pk > 0) { 
+                    console.log("nice!")
```
Comment (aldnav) [inline client/src/pages/Login.vue]: Please refrain from pushing log lines.

File: client/src/pages/Register.vue [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+                    </v-flex>
+                    
+                  </v-layout>
+                  <v-select
+                    v-model="gender"
+                    :items="genderList"
+                    label="Gender"
+                    single-line>
+                  </v-select>
+                  <v-btn color="success" @click="register()">Register</v-btn>
```
Comment (aldnav) [inline client/src/pages/Register.vue]: I think `@click="register"` is enough

Comment (aldnav) [reply client/src/pages/Register.vue]: Unless you intend this to be a complex expression.

File: .gitignore (hunk 2 above)
Comment (boredchinese) [reply .gitignore]: This is weird. I resolved the conflict for the `.gitignore` file but idk why it became like this

File: client/src/pages/Appeal/Create.vue (hunk 4 above)
Comment (boredchinese) [reply client/src/pages/Appeal/Create.vue]: This was supposed to change the "state" of the snackbar component but I only implemented it now.

**Pull request merged**